test/upgrade/upgrade_test.go

```diff
@@ -0,0 +1,79 @@
+// +build upgrade
+
+/*
+Copyright 2020 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package upgrade
+
+import (
+	"testing"
+
+	"go.uber.org/zap"
+	_ "knative.dev/pkg/system/testing"
+	pkgupgrade "knative.dev/pkg/test/upgrade"
+	"knative.dev/serving/test/upgrade/installation"
+)
+
+func TestServingUpgrades(t *testing.T) {
+	c := newUpgradeConfig(t)
+	suite := pkgupgrade.Suite{
+		Tests: pkgupgrade.Tests{
+			PreUpgrade: []pkgupgrade.Operation{
+				ServicePreUpgradeTest(),
+				ServicePreUpgradeAndScaleToZeroTest(),
+				BYORevisionPreUpgradeTest(),
+				InitialScalePreUpgradeTest(),
```

Comment:
Maybe, create additional public methods for those operations. I see there are multiple ones for serving. Having single method like `ServingPreUpgradeTests() []pkgupgrade.Operation` would be much simpler to integrate on `knative/operator` and vendor level.

Reply:
I don't understand. Do you mean wrap all these methods in a new single method that would return
```[]pkgupgrade.Operation{
				ServicePreUpgradeTest(),
				ServicePreUpgradeAndScaleToZeroTest(),
				BYORevisionPreUpgradeTest(),
				InitialScalePreUpgradeTest(),
			}
``` ?

Reply:
Yep. Exaclly.